fhirstore/search/search_methods.py

```diff
@@ -54,16 +59,33 @@ def build_element_query(key, value):
         elif string_modifier == "below":
             element_query["simple_query_string"]["query"] = f"({value})*"
             element_query["simple_query_string"]["fields"] = [string_field]
+        elif string_modifier == "identifier":
+            element_query["simple_query_string"]["query"] = value
+            element_query["simple_query_string"]["fields"] = [
+                f"{string_field}.identifier.value"
+            ]
 
-    elif numeric_modif:
+    elif numeric_prefix:
```

Comment:
small question because I didn't find the info in the documentation: couldn't a query have both prefixes and modifiers?

Reply:
Prefixes are used for ordered parameters like number, date, and quantity. 
Modifiers are used on strings, token, references or uri parameters, which are not ordered. 

The only modifier used on all parameters is :missing, but it is not supported at the moment.

Reply:
Right, thanks!
Do someone have to tell when the user sends an incorrect query (with a modifier on a number for instance)?

Reply:
Yes, there needs to be an operationOutcome error, I'll write an issue for this!